mapfishapp/src/main/webapp/WEB-INF/jsp/index.jsp

```diff
@@ -65,7 +47,7 @@ ResourceBundle bundle = org.georchestra.mapfishapp.ws.Utf8ResourceBundle.getBund
 String detectedLanguage = rLocale.getLanguage();
 String forcedLang = request.getParameter("lang");
 
-String lang = defaultLanguage;
+String lang = request.getParameter("defaultLanguage");
```

Comment:
There's no such parameter to my knowledge

Reply:
It's not the name of a property from a properties file, it's a parameter passed in the Spring configuration file, see https://github.com/georchestra/georchestra/pull/2430/files#diff-c15e9e41a281b1d28fad1be82868b16eR60

Reply:
okay !